src/Uno.UI/UI/Xaml/Documents/Hyperlink.cs

```diff
@@ -24,6 +25,8 @@ public sealed partial class Hyperlink : Span
 		private readonly IFocusable _focusableHelper;
 #endif
 
+		private const string HyperlinkForegroundPressedKey = "HyperlinkForegroundPressed";
+		private const string HyperlinkForegroundPointerOverKey = "HyperlinkForegroundPointerOver";
```

Comment:
These are only defined for Fluent v2. I wonder how this can work when using non-fluent. I see you already have the right test combination, but really can't see how this can work on non-fluent.

Reply:
Oh I see you defined them. Browser's CTRL+F failed me somehow